metis-enrichment/metis-enrichment-rest/src/test/java/eu/europeana/enrichment/rest/OldEnrichmentControllerTest.java

```diff
@@ -30,7 +30,7 @@
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 
-public class EnrichmentControllerTest {
```

Comment:
Would it be easy to copy this test class and adapt it to the new enrichment controller? So that that class is covered by tests as well?

Reply:
That's a good point, I will take care of it